src/main/java/com/patchworkmc/manifest/converter/ModManifestConverter.java

```diff
@@ -105,4 +110,29 @@ private static JsonArray convertAuthorsList(String authors) {
 
 		return array;
 	}
+
+	private static JsonObject getHardDependencies(ModManifest manifest, ModManifestEntry mod) {
```

Comment:
Is it possible to reduce the code duplication between these two functions?

Reply:
Yup!